src/app/shared/filter-bar/filter-bar.component.css

```diff
@@ -1,22 +1,67 @@
-.dropdown-filters .mat-form-field {
+.filters .mat-form-field {
   margin: 8px;
   font-size: 14px;
   max-width: 20%;
   width: 14%; /* depends on number of filters*/
 }
 
-.search-bar {
-  width: 80%;
+ul {
+  margin: 8px;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  list-style-type: none;
```

Comment:
Good substitution for the mat-grid components. Could you enforce a height of 80px so that no vertical scrollbar appears ? This was was enforced by the previous rowHeight property.